http_request_randomizer/requests/proxy/requestProxy.py

```diff
@@ -51,7 +51,10 @@ def __init__(self, web_proxy_list=[], sustain=False):
         self.parsers = parsers
         self.proxy_list = web_proxy_list
         for i in range(len(parsers)):
-            self.proxy_list += parsers[i].parse_proxyList()
+            try:
+                self.proxy_list += parsers[i].parse_proxyList()
+            except ReadTimeout:
+                self.logger.debug("Proxy Parser failed for: {}".format(parsers[i].url))
```

Comment:
These minor changes will do. Also lets mock the Requests in TestProxyProviders class as you suggested.
The changes need to be on the same PR as they tests currently fail!

Thanks :)